package.json

```diff
@@ -13,6 +13,7 @@
   },
   "dependencies": {
     "@headlessui/react": "^1.7.18",
+    "@octokit/graphql": "^8.0.1",
```

Comment:
Why was this package added? The default octokit package which we already have itself can dispatch graphql queries right? And this package doesn't seems to be used in the PR you've made either.